internal/config/nri/nri.go

```diff
@@ -0,0 +1,57 @@
+package nri
+
+import (
+	nri "github.com/containerd/nri/v2alpha1/pkg/runtime"
+	"github.com/pkg/errors"
+)
+
+// Config represents the CRI-O NRI configuration.
+type Config struct {
+	Enabled    bool   `toml:"enable_nri"`
+	ConfigPath string `toml:"nri_config_file"`
+	SocketPath string `toml:"nri_socket_path"`
+	PluginPath string `toml:"nri_plugin_dir"`
```

Comment:
Are those configuration variables actually being used?

Reply:
They were not passed to NRI. Fixed.